app/Bookstore.Domain/Addresses/Address.cs

```diff
@@ -34,6 +35,8 @@ public Address(Customer customer, string addressLine1, string? addressLine2, str
         public string ZipCode { get; set; }
 
         public int CustomerId { get; set; }
+
+        [JsonIgnore]
```

Comment:
Is it possible to configure this in code, rather than using an attribute? It would be good to avoid cross-cutting attributes in the domain classes if possible.

Reply:
Good point. Restructured the log messages and removed the jsonignore attributes.